owa/app/index.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {addLocaleData} from 'react-intl';
 import App from './components/app';
 import {BrowserRouter} from 'react-router-dom';
 
+import en from 'react-intl/locale-data/en';
+import es from 'react-intl/locale-data/es';
+import fr from 'react-intl/locale-data/fr';
+import it from 'react-intl/locale-data/it';
+import pt from 'react-intl/locale-data/pt';
+addLocaleData([...en, ...es, ...fr, ...it, ...pt]);
```

Comment:
So this kind of piece of code will be part of what must be changed in the JIRA ticket to come, correct?

Reply:
Yes, this would be tricky to import dynamically.

Reply:
In the JIRA ticket to come, we will advocate for not using those `react-intl` libraries, but rather to have the `Session` REST endpoints return complete locales descriptions.